### PR TITLE
Issue #15340: created InputFormatted file for section 3.3.2 No Line Wrapping

### PR DESCRIPTION
**Pull request by Zopsss**
#15340 

this section's input file did not had much code and most of the imports were unused to formatter removed them in InputFormatted file

### CODE REVIEW

File: src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/InputNoLineWrapping.java [hunk 1]
```diff
@@ -29,4 +29,6 @@ public void fooMethod() {
         1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 12,
         13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24
       };
+
+  PackageStatementTest packageStatementTest = new PackageStatementTest(); // ok
```
Comment (Zopsss) [inline src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/InputNoLineWrapping.java:33]: input had imported this class but never used it, it was imported to show that long import statements are allowed. So I just created an instance of this class for demo purpose

File: src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/InputFormattedNoLineWrapping.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package com.google.checkstyle.test.chapter3filestructure.rule332nolinewrap;
+
+import com.google.checkstyle.test.chapter3filestructure.toolongpackagetotestcoveragegooglesjavastylerule.PackageStatementTest; // ok, long imports are allowed
```
Comment (Zopsss) [inline src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/InputFormattedNoLineWrapping.java:3]: If I this import statement was never used in the class then formatter would have removed it and we would miss and edge case, so I created an instance of it to test the edge case and maintain similarity between both input files

**Review by Zopsss — COMMENTED** [review]
review:

File: src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/InputFormattedNoLineWrapping.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  // --------------------------------------------------------------------------------------------------
+  // violation above 'Line is longer than 100 characters (found 103).'
+
+  private int[] testing =
+      new int[] {
+        1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 12,
+        13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24
+      };
+
+  PackageStatementTest packageStatementTest = new PackageStatementTest(); // ok
```
Comment (Zopsss) [inline src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/InputFormattedNoLineWrapping.java:22]: this class was imported to show that long imports are allowed. The import was unused and formatter would remove unused imports, so to catch this edge case I had to create and instance of this class.

File: src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/InputNoLineWrapping.java (hunk 1 above)
Comment (Zopsss) [inline src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/InputNoLineWrapping.java:33]: created instance in this file also, just to maintain similarity between both input files

**Review by Zopsss — COMMENTED** [review]
review:

**Review by romani — CHANGES_REQUESTED** [review]
item:

File: src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/InputNoLineWrapping.java (hunk 1 above)
Comment (Zopsss) [reply src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/InputNoLineWrapping.java]: done

File: src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/InputFormattedNoLineWrapping.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (Zopsss) [reply src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/InputFormattedNoLineWrapping.java]: done

**Review by romani — APPROVED** [review]
Ok to merge

**Comment by romani** [comment]
CI failure can be ignored, unrelated.
We should fix it in separate PR.

**Review by rdiachenko — APPROVED** [review]
lgtm

**Pull request merged**